databricks-iris/{{ cookiecutter.repo_name }}/conf/base/logging.yml

```diff
@@ -7,26 +7,39 @@ formatters:
     format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
 
 handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: simple
+    stream: ext://sys.stdout
+
   info_file_handler:
     class: logging.handlers.RotatingFileHandler
     level: INFO
     formatter: simple
-    filename: /dbfs/FileStore/iris-databricks/info.log
+    filename: info.log
```

Comment:
Any reason for this change?

Reply:
Yes, this was a workaround for projects deployed as packages via jobs as the default rich logging does not work for them (see screenshot below).

This strategy of writing logs to DBFS actually came with its own set of problems, as, while the logs would be written correctly, `stdout` would also be flooded with logging errors as `RotatingFileHandler` is not entirely compatible with DBFS. Specifically, some methods that Python's file objects use (`seek` and `flush`) aren't supported on DBFS.

I would rather have `console` logging (which works in both notebooks and in jobs) as the default and would discourage the use of this strategy, as the logging errors could overwhelm users and become unpredictable in the future. I only used it originally as I thought `console` logging also did not work with jobs either, though I have since figured out it does! (See screenshots below).

Reply:
That make senses! I guess it's not a very important feature so I am fine with using `console` instead.